patches/api/0270-EntityMoveEvent.patch

```diff
@@ -18,10 +18,13 @@ index 0000000000000000000000000000000000000000..fae0b1daf733e85d8f691f5ef934b8c0
 +import org.bukkit.event.Cancellable;
 +import org.bukkit.event.HandlerList;
 +import org.bukkit.event.entity.EntityEvent;
++import org.bukkit.event.player.PlayerMoveEvent;
 +import org.jetbrains.annotations.NotNull;
 +
 +/**
 + * Holds information for living entity movement events
++ * <p>
++ * Does not fire for players, please use {@link PlayerMoveEvent} instead.
```

Comment:
this *could* have a `<!-- Paper start -->` and end, though i think some of paper's javadoc patches don't add Paper comments so 🤷🏻‍♀️

Reply:
Probably also not necessary since this is a paper event class.

Reply:
In general, we don't care all that much about javadocs. They're not the most important thing in the world, and they only _really_ affect Paper's site.